### PR TITLE
Models

### PR DESCRIPTION
**Pull request by tinoilcotechino**
This PR adds models and their tests

### CODE REVIEW

File: mobilizon_bots/storage/db.py [hunk 1]
```diff
@@ -1,25 +1,40 @@
...
         )
-        if not self.is_init():
-            # Generate the schema
+        if not self.is_init:
             await Tortoise.generate_schemas()
-
-    def is_init(self) -> bool:
-        # TODO: Check if DB is openable/"queriable"
-        return self.path.exists() and (not self.path.is_dir())
+            logger.info(f"Succesfully initiated database at {self.path}")
```
Comment (chobeat) [inline mobilizon_bots/storage/db.py:35]: initialized

File: mobilizon_bots/models/notification.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+from tortoise import fields
+from tortoise.models import Model
+
+
+class Notification(Model):
+    id = fields.UUIDField(pk=True)
+    # TODO: Make actual IntEnumField
+    status = fields.IntField()
```
Comment (chobeat) [inline mobilizon_bots/models/notification.py:8]: I think this one should be addressed in this PR. We saw it's part of Tortoise anyway, no?

File: mobilizon_bots/models/event.py [hunk 3]
```diff
@@ -3,16 +3,27 @@
...
+    mobilizon_id = fields.TextField()
+    mobilizon_link = fields.TextField()
+    thumbnail_link = fields.TextField(null=True)
+
+    location = fields.TextField(null=True)
+
+    begin_datetime = fields.DatetimeField()
+    end_datetime = fields.DatetimeField()
+    # UTC offset in seconds
+    utcoffset = fields.IntField()
```
Comment (chobeat) [inline mobilizon_bots/models/event.py:19]: why is this necessary? Doesn't "DateTimeField" already store the timezone? Also what does this refer to?

**Review by chobeat — CHANGES_REQUESTED** [review]

File: mobilizon_bots/models/event.py (hunk 3 above)
Comment (tinoilcotechino) [reply mobilizon_bots/models/event.py:19]: I think this is best discussed synchronously. Tortoise does support timezones but you can either set a global timezone that'll be used when selecting datetimes or just store everything as UTC. 

https://tortoise-orm.readthedocs.io/en/latest/timezone.html

I believe the second is the most flexible solution as it'll allow us to store different events with different timezones (but I'm not sure about actual use cases for different timezones)

File: mobilizon_bots/storage/db.py (hunk 1 above)
Comment (tinoilcotechino) [reply mobilizon_bots/storage/db.py:35]: Thanks, fixed

File: mobilizon_bots/models/notification.py (hunk 2 above)
Comment (tinoilcotechino) [reply mobilizon_bots/models/notification.py:8]: Yes, I also wanted to discuss the Notification model synchronously as last time we didn't state clearly the scope/requirements of the model. Anyway I'll try to set up a proposal before our next meeting.
